appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# https://github.com/sindresorhus/appveyor-node/blob/master/appveyor.yml
+
+environment:
+  matrix:
+    - platform: x64
+
+image: Visual Studio 2015
```

Comment:
@Tyriar looks like `Install-Product node 6 x64` [is enough](https://twitter.com/appveyor/status/793176028772302848) 😅 I'll try to remove it later. Thanks for your concern!